src/storage/filters/jsonQuery.ts

```diff
@@ -22,7 +22,8 @@ export type FilterOperatorMethod =
   | '$ne'
   | '$gte'
   | '$lte'
-  | '$sd';
+  | '$sd'
```

Comment:
do we need to add between / nonbetween operators here

Reply:
yes, added